test/classes/diten/multipledestructor.chpl

```diff
@@ -1,7 +1,7 @@
 class C {
-  proc ~C() { writeln("In ~C #1"); }
+  proc deinit() { writeln("In deinit #1"); }
 }
-proc C.~C() { writeln("In ~C #2"); }
+proc C.deinit() { writeln("In deinit #2"); }
```

Comment:
For my own sanity, I would prefer a copy of this test be made in either the test/classes/deinitializers or test/classes/initializers/ directories, rather than overwriting a test for destructors.  It allows me to limit the spot check testing I do to just those directories rather than running over the entire test/classes directory structure (which gets covered by my full standard run before committing anything)